### ISSUE TITLE
Assess how each WCAG 2.2 criterion may affect existing styles, components and patterns

### ISSUE THREAD

**Issue by davidc-gds** [alphagov/govuk-frontend#3177]
## What

Using an [internal team spreadsheet](https://docs.google.com/spreadsheets/d/194aFvTTi4JlyEw4b21dHmCn5Iry4t4RY8A8VOKydFj0/edit?usp=sharing), we'll assess all the current styles, components and patterns published within the GOV.UK Design System, based on their estimated likelihood of:

- having one of the 9 new WCAG 2.2 criterion be applicable
- being at risk of a WCAG failure under the 9 new WCAG 2.2 criterion

## Why
As we approach the publication of the WCAG 2.2 recommendation, the GOV.UK Design System needs to prepare to make updates across all of our styles, components and patters, to align them with the new criteria.

This work supports an epic from our accessibility activity log: [Update styles, components and patterns to WCAG 2.2 AA (#3040)](https://github.com/alphagov/govuk-frontend/issues/3040).

This assessment will allow us to:

- Assess our overall risk of potential upcoming WCAG 2.2 failures for design system styles, components and patterns
- Help us prioritise work, so we focus on the most pressing, complex, and important accessibility updates first
- Measure overall progress for our update to WCAG 2.2 compliance

## Who needs to work on this
Accessibility specialist

## Who needs to review this
Developers
Product manager
Delivery manager

## Done when

- [x] Learn about each of the [9 added or modified criteria in the WCAG 2.2 candidate recommendation](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22)
- [x] Determine the applicability of each WCAG 2.2 criterion for each style. component and pattern
- [x] Determine the risk that each style, component and pattern may lead to a failure for each WCAG 2.2 criterion
- [x] Rank styles, components and patterns, based on their applicability and risk pertaining to all WCAG 2.2 criterion
- [x] Categorise each entry by the work type (content, development, investigation, accessibility research)
- [x] Share the results with the team, with the intent to prioritise style, component and pattern updates
- [x] Publish the results in a publicly available format (potentially a GitHub Project, spreadsheet, or table)

**Comment by davidc-gds** [comment]
I spoke to @stevenjmesser on Friday, and we're thinking sharing the source spreadsheet for our WCAG 2.2 element assessments might lead to confusion or misinterpretation by service teams. The spreadsheet is really only valuable as a prioritisation tool for us.

For that reason, we'll likely modify the final 'done when' item for this issue: "Publish the results in a publicly available format (potentially a GitHub Project, spreadsheet, or table)"

Instead, we may now share the information publicly in a few alternative ways:

- The [survey tool we're developing for service teams](https://github.com/alphagov/design-system-team-internal/issues/663), to help them determine where they'll need to focus their efforts after updating to the 'supports WCAG 2.2' version of govuk-frontend
- A less visible but publicly available [GitHub issue where we track all the WCAG 2.2 issues we create](https://github.com/alphagov/govuk-design-system/issues/2654) based on the entries in the assessment spreadsheet
- the 'WCAG 2.2' tag in GitHub ([govuk-frontend tag](https://github.com/alphagov/govuk-frontend/labels/WCAG%202.2), [design system tag](https://github.com/alphagov/govuk-design-system/labels/WCAG%202.2))
